apps/ledger-live-mobile/e2e/specs/swap/dexSwap.spec.ts

```diff
@@ -24,9 +24,9 @@ describe("DEX Swap", () => {
 
```

Comment:
(side comment from this review): this folder seems to have assigned hub and qa as owners. i'm not even sure what we want for them, but i feel swap/ could be owned by ptx team? ( cc @KVNLS @aboissiere-ledger ) – in these case, let's always use another PR to amend the codeowners